papermill/tests/test_iorw.py

```diff
@@ -78,12 +79,28 @@ def test_read(self):
             self.papermill_io.read("fake/path"), "contents from fake/path for version 1"
         )
 
+    def test_read_with_no_file_extension(self):
+        with pytest.warns(UserWarning):
+            self.papermill_io.read("fake/path")
+
+    def test_read_with_invalid_file_extension(self):
```

Comment:
@harsham4026 You needed both `-W` and `always` so that it reads as `pytest -v --maxfail=2 --cov=papermill --pyargs papermill -W always`. Unless I missed a commit's changes, you only added the `-W` part

Reply:
@MSeal thank you. Added the part -W always you suggested and now the build is successful.